lib/semian/lru_cache.rb

```diff
@@ -0,0 +1,54 @@
+class LRUCache
+  extend Forwardable
+  def_delegators :@table, :size, :count
+  def_delegators :@table, :empty?
+  def_delegators :@table, :keys
+  def_delegators :@table, :clear
+
+  attr_accessor :table, :minimum_time_in_queue
```

Comment:
Do you really want people to change these two at runtime? They seem like they should be `attr_reader`, no?